### DISCUSSION TITLE
Usage of "greeting" and "salutation" on `SimpleMessage` is grammatically incorrect

### DISCUSSION THREAD

**Discussion by shengslogar** [laravel/framework#48951] [General]
I'll be the first to admit this is overly pedantic, and likely not worth serious consideration for the sake of backwards-compatibility, but thought this would be interesting to note.

Laravel uses "greeting" to open a message, and "salutation" to close it. However, both of these terms are largely synonymous. Merriam Webster defines "greeting" as ["an expression of good wishes"](https://www.merriam-webster.com/dictionary/greeting) and "salutation" as ["the word or phrase of greeting... that conventionally comes immediately before the body of a letter"](https://www.merriam-webster.com/dictionary/salutation).

If a PR were created for this, I would keep "greeting" for opening the message (it's less formal and more widely understood), and choose something like "closing" to replace "salutation".

Excuse me while I go do something more productive with my life. 😆 

**Related code:**

https://github.com/laravel/framework/blob/fc8f819bef5b4843baadda393c51320ceeb41a70/src/Illuminate/Notifications/Messages/SimpleMessage.php#L24-L36

https://github.com/laravel/framework/blob/fc8f819bef5b4843baadda393c51320ceeb41a70/src/Illuminate/Notifications/resources/views/email.blade.php#L2-L11

https://github.com/laravel/framework/blob/fc8f819bef5b4843baadda393c51320ceeb41a70/src/Illuminate/Notifications/resources/views/email.blade.php#L38-L44

**Comment by Healyhatman** [comment]
I would, at the very least, comment the function so it actually says that it's a closing message. "Sets the salutation" tells me only as much as the function name does.

**Comment by craigrileyuk** [comment]
I just got these mixed up too. Very confusing.
